tests/functional/raw-node/utils/makeRequest.js

```diff
@@ -60,6 +62,26 @@ function makeRequest(params, callback) {
         rejectUnauthorized: false,
     };
 
+    if (queryObj) {
```

Comment:
Looks like this is being repeated below (https://github.com/scality/S3/pull/1078/files#diff-811e22af2f7d03b99d66636c51d1ad5eR128), causing extra query strings in the test request.